### PR TITLE
add Vue (Nuxt.js 3)  example

### PR DESCRIPTION
**Pull request by KATT**
>  Currently blocked by #1129 

Adds Nuxt.js 3 example.

## Todo

- [x] Fix `yarn dev`
- [x] Fix `yarn build`
- [x] Get rid of [ugly long `import`](https://github.com/trpc/trpc/pull/1124/files/d0e0ea1d138f52fbe125459474a7a6aaaa322d69#diff-ca4c9db10cf79546f3a70276cfe6dfb5ad90b231961b652190e6dc3553a22c0c)
- [x] ❓ `.vscode/` recommendations?
- [ ] Fix ESModules <-> CommonJS interop - #1129
- [ ] ❓ `/post/[id]` detail page?
- [ ] ❓ Add Prisma & superjson?

## How-to try it out

For anyone who'd like to help:

```bash
git clone git@github.com:trpc/trpc.git
cd trpc
git checkout katt/nuxt
yarn
cd examples/nuxt
yarn dev
```

### CODE REVIEW

**Comment by KATT** [comment]
@all-contributors add @danielroe for code

**Comment by KATT** [comment]
@danielroe the project will automatically be mirrored in https://github.com/trpc/examples-nuxt similar to [this](https://github.com/trpc/examples-next-prisma-starter) - so feel free to add any `./vscode` folder within the actual example dir too :)

**Comment by KATT** [comment]
Feel free to add a new "Vuxt"-page in the docs (`/www`-folder) - can be done as a follow-up as well.

File: README.md [hunk 1]
```diff
@@ -61,12 +63,21 @@ tRPC allows you to easily build & consume fully typesafe APIs, without schemas o
...
 
 ```sh
 npx create-next-app --example https://github.com/trpc/trpc --example-path examples/next-prisma-starter trpc-prisma-starter
 ```
 
+### Nuxt
+
+**Quick start with a full-stack Nuxt example:**
+```sh
+npx nuxi init tprc --template trpc/trpc/examples/nuxt
```
Comment (Eckhardt-D) [inline README.md:78]: @danielroe Small typo? Perhaps `npx nuxi init trpc`?

**Comment by KATT** [comment]
Could be picked up again with the `next` branch as a base

**Pull request closed without merging**